sycl/include/sycl/ext/xilinx/fpga/static_unroll.hpp

```diff
@@ -0,0 +1,157 @@
+//==- static_unroll.hpp --- SYCL templated loop unrolling       -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "CL/sycl/detail/defines.hpp"
+
+#ifndef SYCL_XILINX_FPGA_PARALLEL_INVOKE_HPP
+#define SYCL_XILINX_FPGA_PARALLEL_INVOKE_HPP
+
+__SYCL_INLINE_NAMESPACE(cl) {
+
+namespace sycl {
+
+namespace ext::xilinx {
+namespace detail {
+
+///
+///@brief partial_static_unroll implementation
+///
+///@tparam NormalizedInitStep Normalized unrolled iteration index
```

Comment:
Good to see the renaming and all the comments.
But I do not understand this one. "start" or "initial" should appear somewhere.

Reply:
Perhaps an example would help :-)(